inlong-sort/sort-core/src/test/java/org/apache/inlong/sort/parser/MySqlTemporalJoinRelationSqlParseTest.java

```diff
@@ -0,0 +1,286 @@
+        if (left) {
+            return new LeftOuterTemporalJoinRelationRelation(inputIds, outputIds, conditionMap, systemTime);
+        }
+        return new InnerTemporalJoinRelationRelation(inputIds, outputIds, conditionMap, systemTime);
+    }
+
+    /**
+     * Test inner temporal join with event time for extract is mysql {@link MySqlExtractNode},
+     * {@link KafkaExtractNode} and load is mysql {@link MySqlLoadNode}
+     *
+     * @throws Exception The exception may be thrown when executing
+     */
+    @Test
+    public void testInnerTemporalJoinWithEventTimeParse() throws Exception {
+        StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
+        env.setParallelism(1);
+        env.enableCheckpointing(10000);
+        env.disableOperatorChaining();
+        EnvironmentSettings settings = EnvironmentSettings
+                .newInstance()
+                .useBlinkPlanner()
+                .inStreamingMode()
+                .build();
+        StreamTableEnvironment tableEnv = StreamTableEnvironment.create(env, settings);
+        Node kafkaExtractNode = buildKafkaExtractNode();
+        Node mySqlExtractNode2 = buildMySQLExtractNode();
+        Node kafkaLoadNode = buildKafkaLoadNode();
+        StreamInfo streamInfo = new StreamInfo("1",
+                Arrays.asList(kafkaExtractNode, mySqlExtractNode2, kafkaLoadNode),
+                Collections.singletonList(
+                        buildNodeRelation(Arrays.asList(kafkaExtractNode, mySqlExtractNode2),
+                                Collections.singletonList(kafkaLoadNode), false, new FieldInfo("order_time")))
+        );
+        GroupInfo groupInfo = new GroupInfo("1", Collections.singletonList(streamInfo));
+        FlinkSqlParser parser = FlinkSqlParser.getInstance(tableEnv, groupInfo);
+        ParseResult result = parser.parse();
+        Assert.assertTrue(result.tryExecute());
+    }
+
+    /**
+     * Test left temporal join with event time for extract is mysql {@link MySqlExtractNode},
+     * {@link KafkaExtractNode} and load is mysql {@link MySqlLoadNode}
+     *
+     * @throws Exception The exception may be thrown when executing
+     */
+    @Test
+    public void testLeftTemporalJoinWithEventTimeParse() throws Exception {
+        StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
+        env.setParallelism(1);
+        env.enableCheckpointing(10000);
+        env.disableOperatorChaining();
+        EnvironmentSettings settings = EnvironmentSettings
+                .newInstance()
+                .useBlinkPlanner()
+                .inStreamingMode()
+                .build();
+        StreamTableEnvironment tableEnv = StreamTableEnvironment.create(env, settings);
+        Node kafkaExtractNode = buildKafkaExtractNode();
+        Node mySqlExtractNode2 = buildMySQLExtractNode();
+        Node kafkaLoadNode = buildKafkaLoadNode();
```

Comment:
load node is kafkaLoadNode, but comment describe is mysqlLoadNode

Reply:
done